.changeset/tricky-lamps-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You should not see credential error when running playwright tests
```

Comment:
it's rather: _An error is no longer raised and visible to the user when saving credential is not supported_